content/en/docs/ops/best-practices/security/index.md

```diff
@@ -129,13 +129,72 @@ For example, if you choose the `DECODE_AND_MERGE_SLASHES` option, you modify the
           ...
     {{< /text >}}
 
-### Less common normalization configurations
+### Mitigation for unsupported normalization
 
-#### Case Normalization
+This section describes various mitigation for unsupported normalization. These could be useful when you find a specific
+normalization that is not supported by Istio.
+
+Please make sure you understand the mitigation thoroughly and use it carefully. Some mitigation is just best-effort workaround
+that is not well tested or rely on things that are not supported by Istio.
+
+#### Safer Authorization Policy Pattern
```

Comment:
I suggest moving this up and make the "Apply default-deny authorization policies" a subsection of it:

## Authorization policies

### Safer Authorization Policy Pattern

#### Use default-deny authorization policies
(Content from https://istio.io/latest/docs/ops/best-practices/security/#apply-default-deny-authorization-policies)

#### Use `ALLOW-with-positive-matching` and `DENY-with-negative-match` patterns
(Content from this section)

Reply:
sounds good.